src/Book.java

```diff
@@ -0,0 +1,29 @@
+public class Book {
+    String name;
+    String authorBook;
```

Comment:
Не тот тип. Автор это у нас автор, а не строка)
```suggestion
    Author authorBook;
```